stanford_profile_helper.module

```diff
@@ -603,3 +603,34 @@ function stanford_profile_helper_preprocess_pattern_localfooter(&$variables) {
   }
 
 }
+
+/**
+ * Implements hook_form_FORM_ID_alter().
+ */
+function stanford_profile_helper_form_media_embeddable_add_form_alter(array &$form, FormStateInterface $form_state) {
+  _stanford_profile_helper_embeddable_field_access($form, $form_state);
+}
+
+/**
+ * Implements hook_form_FORM_ID_alter().
+ */
+function stanford_profile_helper_form_media_embeddable_edit_form_alter(array &$form, FormStateInterface $form_state) {
+  _stanford_profile_helper_embeddable_field_access($form, $form_state);
+}
+
+/**
+ * Add a field permission to the open embed code field.
+ *
+ * @param array $form
+ *   The form array.
+ * @param \Drupal\Core\Form\FormStateInterface $form_state
+ *   The form state.
+ */
+function _stanford_profile_helper_embeddable_field_access(array &$form, FormStateInterface $form_state) {
```

Comment:
This isn't getting called when in the media library from the wysiwyg

Reply:
FINE!